src/types/Sort.ts

```diff
@@ -0,0 +1,5 @@
+export enum Sort {
+  all = 'All',
+  active = 'Active',
+  completed = 'Completed',
```

Comment:
```suggestion
export enum Sort {
  All = 'all',
  Active = 'active',
  Completed = 'completed',
```
Enum properties should be named in PascalCase